### PR TITLE
add landing route.tsx

### PR DESCRIPTION
**Pull request by hkcj**
I believe this is request #21

### CODE REVIEW

File: src/landing/moudles/Route.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Switch, Route } from "react-router-dom";
+import Main from "./Main";
+
+const Routes = () => {
+  <Switch>
+    <Route path="/" exact component={Main} />
```
Comment (zt1983811) [inline src/landing/moudles/Route.tsx]: Must: Are you sure this is pointing to Main component? I thought we would like to point to login component

**Comment by hkcj** [comment]
Isn't this Main the login page?

**Comment by zt1983811** [comment]
> Isn't this Main the login page?

I thought we have agreed to move Login to separate component and name it Login.tsx. Main.tsx will only contains the layout of everything for landing moudle

**Comment by hkcj** [comment]
Hi Tong,
yes, I remembered. 
I've completed it actually but just afraid it's still not right.
I'm going to push it in a jiffy.

**Comment by zt1983811** [comment]
> Hi Tong,
> yes, I remembered.
> I've completed it actually but just afraid it's still not right.
> I'm going to push it in a jiffy.

I just checked it looks perfect. I will approve it. Could you merge it please?
I have approved it, to merge you just need to click the green button called Merge Pull Request

**Review by zt1983811 — APPROVED** [review]

**Pull request merged**